source/ore/publish.rst

```diff
@@ -145,3 +145,57 @@ Support
 -------
 
 Support cannot do anything, merely a way of showing that the member is a part of the project.
+
+
+Documenting Your Project With Pages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To document your plugin Ore offers the creation of pages, by default you get a 'Home' page when you create your new Project.
+To add new pages you can click on the '+' icon in the Pages bar on the right of the screen, doing so will open a pop-up where you 
+can choose a name and the parent page (adding a page on the Home page is not possible). If you choose the '<None>' option the page 
+will become a root page where you can later add child pages on, if you hover choose to add it to an other page the page will only show
+when the parent page is expanded.
+To document you pages you can use CommonMark.
+
+Linking pages
+-------------
+
+If you have the following tree structure in you pages:
+
+- Home
+- Commands
+    - User
+    - Admin
+- Setup
+    - Config
+    - Permissions
+
+And you want to add a link from the home page to the Config page in Setup you can use `WikiLinks`.
+To add a WikiLink use the format `[[Link]]` so on the home page you can add `[[Setup/Config]]` and this will get be converted to a
+link that is guaranteed to work.
+
+.. note::
+    You can also specify a title for the link and separate it with a Pipe symbol so `[[Title|Link]]` will become a link with the title
+
+.. note::
+    WikiLinks are only supported in Ore so if you use them on the Home-page they will not be displayed correctly on the forums.
+    However, we still recommend using WikiLinks instead of normal links to guarantee the future working of your wiki.
+
+Project states
+~~~~~~~~~~~~~~~
+
+When creating a new project a banner will appear telling you your project is in a 'new' state.
+The new state is intended to indicated to users and admins you are still working on the setup of your project. If you are done documenting 
+your plugin to make it comply with the guidelines you can click the Button 'Publish' to mark you are done.
+If you however don't Publish it the project will be published after 24 hours since you created the project.
+
+Needs Changes
+-------------
+
+Ore staff can hide your project and request changes the changes in question will be listed in the banner. This state will hide your project from the 
```

Comment:
Semicolons needed between "... changes; the changes ..."  and also
"This state will hide your project from the public;  if you are done ..."